build-rpi-raspbian/first-boot.sh

```diff
@@ -41,7 +41,23 @@ echo "watch cat /boot/first-boot.log" > "/home/$username/.bash_profile"
 if [ -z "${wifi_ssid}" ]; then
   echo "$(timestamp) [openHABian] Setting up Ethernet connection... OK"
 elif grep -q "openHABian" /etc/wpa_supplicant/wpa_supplicant.conf; then
-  echo "$(timestamp) [openHABian] Setting up Wi-Fi connection... OK"
+  echo -n "$(timestamp) [openHABian] Setting up Wi-Fi connection... "
+  if [ ! -z "$(iwlist wlan0 scanning 2>&1 | grep "Interface doesn't support scanning")" ]; then
```

Comment:
I have added the test for the symptom using established tools rather than executing rfkill that might not be present.

Reply:
I have tested the execution of these scenarios on my rp3b+

Reply:
You can actually reduce this one nicely to it's essence:

```shell
if iwlist wlan0 scanning 2>&1 | grep -q "Interface doesn't support scanning"; then
```